src/promise/Promise.php

```diff
@@ -59,19 +59,19 @@ public function isResolved() : bool{
 	 * will be an array containing the resolution values of each Promises in
 	 * `$promises` indexed by the respective Promises' array keys.
 	 *
-	 * @phpstan-param Promise<mixed>[] $promises
+	 * @phpstan-template TTValue
```

Comment:
`TTValue`?

Reply:
Phpstan noted that TValue conflicted with the Promise template. Could this be an isolated problem with my local configuration, idk

Reply:
No, it's correct. Seems like this could be a problem though, as there's no way for it to infer template types and it will likely complain.